tests/integration/test_storage_s3_queue/test.py

```diff
@@ -734,7 +734,7 @@ def test_multiple_tables_streaming_sync_distributed(started_cluster, mode):
         create_mv(instance, table_name, dst_table_name)
 
     total_values = generate_random_files(
-        started_cluster, files_path, files_to_generate, row_num=1
+        started_cluster, files_path, files_to_generate, row_num=50
```

Comment:
why?

Reply:
This is a test where clickhouse servers on different nodes process the same files in parallel and there is a check in the test that both servers made progress and not just one server processed all files. But sometimes this check fails. I saw by the logs that second server was just unable to keeper up as files were quite small (1 row per file), so 1st server was downloading the file and starting the next file quicker than second server checked for current file state and moved to the next. (Also the fact that MV is created not precisely at the same time for each server, but one after another gives the first server some speed up). So I decided to increase the number of rows per file to make more chance for second server to keep up.